Fix amibguous sentence in Scope-section.

june128 opened the pull request:
The first sentence of the Scope-section didn't clearly indicate that the `when an individual is representing the project or its community`-part only applies to the second half of the sentence (`in public spaces`) and not the whole sentence.
This PR therefore changes the layout of the sentence to remove all ambiguity; as discussed here: https://github.com/ContributorCovenant/contributor_covenant/issues/659

june128 committed:
Fix amibguous sentence in Scope-section.

The first sentence of the Scope-section didn't clearly indicate that the
"when an individual is representing the project or its community"-part
only applies to the second half of the sentence ("in public spaces") and
not the whole sentence.
Therefore change the layout of the sentence to remove all ambiguity.

For more info, see:
https://github.com/ContributorCovenant/contributor_covenant/issues/659

Co-Authored-By: Coraline Ada Ehmke <coraline@idolhands.com>

julianschacher commented:
@CoralineAda I entered you as a Co-Author for the commit (8cd4c56c7b175881e4f8bd1a69f33f287f7919e6), since you suggested the wording. I hope that's ok.

june128 committed:
Add "it" to fix grammar.

Someone pointed out in a conversation, that the sentence is
grammatically incorrect and that adding the "it" would fix it.

CoralineAda commented:
Great work, thank you!
